helpers/account_helper.py

```diff
@@ -0,0 +1,94 @@
+from json import loads
+from requests import Response
+
+from services.dm_api_account import DMApiAccount
+from services.api_mailhog import MailHogApi
+
+
+class AccountHelper:
+
+    def __init__(self, dm_account_api: DMApiAccount, mailhog: MailHogApi):
+        self.dm_account_api = dm_account_api
+        self.mailhog = mailhog
+
+    def register_new_user(self, login: str, password: str, email: str):
+        json_data = {
+            'login': login,
+            'email': email,
+            'password': password
+        }
+
+        response = self.dm_account_api.account_api.post_v1_account(json_data=json_data)
+        assert response.status_code == 201, f"Пользователь {login} не был создан"
+
+        token = self.get_activate_token_by_login(response=response, login=login)
```

Comment:
Добавил комментарий ниже 
```suggestion
        token = self.get_activate_token_by_login(login=login)
```